### DISCUSSION TITLE
[solved] Aqara contact sensor - battery/temp sensor not working

### DISCUSSION THREAD

**Discussion by ghost** [Koenkk/zigbee2mqtt#12261] [Devices]
Hi,

I wrote this as a discussion (not a bug) as my issue maybe due to my not running the latest Sonoff 3.0 Zigbee USB dongle firmware.

I have just got everything working for the first time.  Everything is on the same RPi running DietPi OS.
- Homeassistant core-2022.4.6 installed via _dietpi-software_
- Mosquito installed via _dietpi-Software_
- MQTT integration installed within HA
- Zigbee2MQTT running in docker
- Sonoff Dongle has zStack3x0  v20210708 (the one it came installed with).  I have a Mac (not Windows PC) and cannot work out out how to flash the latest version to the Sonoff.

Everything is working except for the temp and battery sensors for the Aqara contact sensor (model MCCGQ11LM).  The battery sensor worked fine when I was using the Sonoff E1 hub through HA.

Can anyone give me a pointer?  ...is it the Sonoff firmware?

**Comment by pedroke** [comment]
Have you checked in Home assistant, maybe the entities for them are disabled. Go to the device and there you should see its entities and also the list of those disabled. Some of them are disabled by default but can be enabled.

**Comment by ghost** [comment]
Hi,

Thanks for the reply.  No, they are not disabled in HA.

Iwasn't getting a value for them in the ZIgbe2MQTT dashboard either.

**Comment by ghost** [comment]
Huh.  I have just noticed that they are both now showing.

Perhaps it just takes a little while for the values to be broadcast?

**Reply by pedroke** [reply]
Yes it takes. I didn't care that much about temperature, but battery/voltage always takes some time and it is not reported from the very begining.

**Comment by ghost** [comment]
cool.  I Alsop don't care about temperature but have an automation to alert me when batteries are low

thanks!
